new_platforms/include/oehost.h

```diff
@@ -24,7 +24,7 @@ extern "C" {
 typedef intptr_t ssize_t;
 #include <openenclave/bits/result.h>
 
-#ifndef __in_ecount
+#ifndef _MSC_VER
```

Comment:
oeenclave.h line 29 has instead:
#if !defined(_MSC_VER) || defined(OE_NO_SAL)
why are oehost.h and oeenclave.h different here?